core/helpers/EEH_Money.helper.php

```diff
@@ -159,79 +159,88 @@ public static function compare_floats($float1, $float2, $operator = '=')
     /**
      * This returns a localized format string suitable for jQplot.
      *
-     * @param string $CNT_ISO  If this is provided, then will attempt to get the currency settings for the country.
-     *                         Otherwise will use currency settings for current active country on site.
+     * @param string|null $CNT_ISO If this is provided, then will attempt to get the currency settings for the country.
+     *                             Otherwise will use currency settings for current active country on site.
      * @return string
      * @throws EE_Error
+     * @throws ReflectionException
      */
-    public static function get_format_for_jqplot($CNT_ISO = '')
+    public static function get_format_for_jqplot(?string $CNT_ISO = ''): string
     {
         // default format
-        $format = 'f';
-        $currency_config = $currency_config = EEH_Money::get_currency_config($CNT_ISO);
```

Comment:
🤔

Reply:
just making sure the variable was set? 🤷🏻‍♂️